scripts/settings/generate-types.js

```diff
@@ -0,0 +1,49 @@
+require("dotenv").config();
```

Comment:
hmm.. I see this as a tool that you have _for yourself_ so as to not write these types by hand -- but these types shouldn't be automated, mainly because the database _cannot_ control our source code - it should be the other way around

Reply:
_ ... I didn't notice this was in draft -- Ignore me_